packages/visualizations/tests/unit/component/navi-visualizations/table-test.js

```diff
@@ -26,6 +41,13 @@ moduleForComponent('navi-visualizations/table', 'Unit | Component | table', {
     this.register('helper:not-eq', Ember.Helper.helper(()=>{}), {instantiate: false});
     this.register('helper:is-valid-moment', Ember.Helper.helper(()=>{}), {instantiate: false});
     this.register('helper:format-number', Ember.Helper.helper(()=>{}), {instantiate: false});
+
+    setupMock()
+    return getOwner(this).lookup('service:bard-metadata').loadMetadata();
+  },
+
+  afterEach() {
+    teardownMock();
```

Comment:
Are all these changes relevant still?? It looks like no tests were changed here.

Reply:
Yes, it is needed, since we use bard metadata service in table